zulipterminal/model.py

```diff
@@ -2071,3 +2071,17 @@ def poll_for_events(self) -> None:
                         self.controller.raise_exception_in_main_thread(
                             sys.exc_info(), critical=False
                         )
+
+    def show_guest_suffix(self, user_id: int) -> bool:
+        """
+        Returns True if the realm_enable_guest_user_indicator
+        setting is enabled and the user is a guest
+        """
```

Comment:
A docstring is for users of the function, not a comment as to how it works.

If you feel there is a need for a description, beyond the name of the function, then please adjust the docstring - but it should not describe the implementation.